hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -221,6 +221,8 @@ public void setConfigPatternMatcher(ConfigPatternMatcher configPatternMatcher) {
      * @param name property name
      * @return property value
      * @see #setProperty(String, String)
+     * @see <a href="http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#system-properties">
```

Comment:
Seems this can be misleading in time, valid properties can be different between versions. A version agnostic link may fit better. But this PR is a step forward.